configure.py

```diff
@@ -2393,8 +2410,7 @@ def release_date(datestamp):
 if __name__ == '__main__':
     try:
         main()
-    except Exception as e:
-        import traceback
+    except ConfigureError as e:
```

Comment:
Do we really want to catch stuff we are not prepared for? I think everything that is thrown without someone thinking about the consequences should crash the program with stack trace.